### DISCUSSION TITLE
Feature request: Watch > Custom > Releases > Exclude betas

### DISCUSSION THREAD

**Discussion by fulldecent** [community/community#109626] [Repositories]
### Select Topic Area

Product Feedback

### Body

When using GitHub releases you can already make a release as a "pre-release" availability. See:

![Screenshot 2024-02-22 at 19 48 10](https://github.com/community/community/assets/382183/a384c4fe-b28e-4f29-aef0-8b025572c06b)

---

I would like to subscribe to email notification when a project makes releases. But I most always only care about non-pre-release versions.

![Screenshot 2024-02-22 at 19 45 29](https://github.com/community/community/assets/382183/42bca8f8-b7b9-4568-ba2d-97bbdb94d438)

Because I get too much mail. And pre-release software is usually not something I act on.

**Comment by namith1003** [comment]
As per many many discussion threads that i have read through for example,

https://github.com/orgs/community/discussions/29192

it seems it is not possible to do so but seems to be a feature in superior demand at the moment but seems to still even after the years seems to not be available.

Hope this helps answer your question, if anything leave a reply,

A.A Namith Nimlaka

**Reply by fulldecent** [reply]
Thank you for the reference. I'm subscribed to that now and this one https://github.com/orgs/community/discussions/4993
